STDIN and STDOUT of Perl CGI scripts should be binary mode for Windows Perl

fujii opened the pull request:
#### 5a716d72adac16a24a5fa865b2ed6a7a454d6d64
<pre>
STDIN and STDOUT of Perl CGI scripts should be binary mode for Windows Perl
<a href="https://bugs.webkit.org/show_bug.cgi?id=263701">https://bugs.webkit.org/show_bug.cgi?id=263701</a>

Reviewed by Ross Kirsling.

Windows Perl converts &apos;\n&apos; to &apos;\r\n&apos; in text mode. STDIN and STDOUT of
Perl CGI scripts have to be binary mode for Windows Perl. &quot;use CGI&quot;
automatically does it. However, &quot;use CGI&quot; is not recomended nowadays,
and it&apos;s inefficient to &quot;use CGI&quot; just for setting binmode for them.
Added &quot;binmode STDOUT&quot; for Perl CGI scripts not using CGI module.
Also, added &quot;binmode STDIN&quot; for scripts reading STDIN.

* LayoutTests/http/tests/cache/resources/current-time.cgi:
* LayoutTests/http/tests/cache/resources/echo-cacheable.cgi:
* LayoutTests/http/tests/cache/resources/echo-no-store.cgi:
* LayoutTests/http/tests/cache/resources/random-cached-iframe.cgi:
* LayoutTests/http/tests/cache/resources/random-cached.cgi:
* LayoutTests/http/tests/cache/resources/random-max-age.cgi:
* LayoutTests/http/tests/cache/resources/random.cgi:
* LayoutTests/http/tests/cache/resources/subresource-failover-to-network.cgi:
* LayoutTests/http/tests/cookies/resources/clearCookies.cgi:
* LayoutTests/http/tests/cookies/resources/getCookies.cgi:
* LayoutTests/http/tests/cookies/resources/setCookies.cgi:
* LayoutTests/http/tests/history/redirect-200-refresh-0-seconds.pl:
* LayoutTests/http/tests/history/redirect-200-refresh-2-seconds.pl:
* LayoutTests/http/tests/history/resources/redirect-helper.pl:
* LayoutTests/http/tests/incremental/pause-in-script-element.pl:
* LayoutTests/http/tests/incremental/resources/slow-utf8-css.pl:
* LayoutTests/http/tests/incremental/slow-utf8-html.pl:
* LayoutTests/http/tests/incremental/slow-utf8-text.pl:
* LayoutTests/http/tests/incremental/split-hex-entities.pl:
* LayoutTests/http/tests/inspector/network/resources/cors-data.pl:
* LayoutTests/http/tests/links/resources/redirect-helper.pl:
* LayoutTests/http/tests/loading/onload-vs-immediate-refresh.pl:
* LayoutTests/http/tests/loading/resources/404-with-empty-body-no-content-type.cgi:
* LayoutTests/http/tests/loading/resources/404-with-empty-body.cgi:
* LayoutTests/http/tests/local/slow-css-pass.cgi:
* LayoutTests/http/tests/mime/resources/style-with-empty-content-type.cgi:
* LayoutTests/http/tests/misc/BOM-override.pl:
* LayoutTests/http/tests/misc/error404.pl:
* LayoutTests/http/tests/misc/nph-non-utf8-header-name.pl:
* LayoutTests/http/tests/misc/resources/BOM-override-2.pl:
* LayoutTests/http/tests/misc/resources/BOM-override-3.pl:
* LayoutTests/http/tests/misc/resources/image-slow.pl:
* LayoutTests/http/tests/misc/resources/intrinsic-ratio-slow.pl:
* LayoutTests/http/tests/misc/resources/percent-sign-in-form-data-field-name.cgi:
* LayoutTests/http/tests/misc/resources/post-echo-and-notify-done.cgi:
* LayoutTests/http/tests/misc/resources/post-echo.cgi:
* LayoutTests/http/tests/misc/resources/script-slow1.pl:
* LayoutTests/http/tests/misc/resources/script-slow2.pl:
* LayoutTests/http/tests/misc/resources/slow-async-script.cgi:
* LayoutTests/http/tests/misc/resources/slow-defer-script.cgi:
* LayoutTests/http/tests/misc/resources/slow-stylesheet.cgi:
* LayoutTests/http/tests/misc/resources/svg-slow.pl:
* LayoutTests/http/tests/misc/resources/text-refresh.pl:
* LayoutTests/http/tests/misc/resources/uncacheable-script.cgi:
* LayoutTests/http/tests/misc/xmltokenizer-do-not-crash.pl:
* LayoutTests/http/tests/navigation/resources/error404.pl:
* LayoutTests/http/tests/navigation/resources/redirect-cycle-1.pl:
* LayoutTests/http/tests/navigation/resources/redirect-cycle-2.pl:
* LayoutTests/http/tests/navigation/resources/redirect-to-referrer-check.pl:
* LayoutTests/http/tests/navigation/resources/redirect302-metaredirect.pl:
* LayoutTests/http/tests/navigation/resources/redirect302.pl:
* LayoutTests/http/tests/navigation/resources/reload-subframe-content.pl:
* LayoutTests/http/tests/navigation/resources/reload-subframe.pl:
* LayoutTests/http/tests/navigation/resources/response204.pl:
* LayoutTests/http/tests/navigation/resources/slow-resource-1-sec.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin-cacheable.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin-unknown-type.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin.pl:
* LayoutTests/http/tests/plugins/resources/plugin-document-has-focus.pl:
* LayoutTests/http/tests/plugins/resources/slow-resource.pl:
* LayoutTests/http/tests/preload/resources/nph-invalid_resources_from_header.pl:
* LayoutTests/http/tests/resources/post-and-verify-hybrid.cgi:
* LayoutTests/http/tests/resources/post-and-verify.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/nph-cached-xfo.pl:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-allowall.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-deny.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-frame-ancestors-same-origin-allow.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-frame-ancestors-same-origin-deny.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-invalid.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-multiple-headers-conflict.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-multiple-headers-sameorigin.cgi:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/frame-ancestors-self-x-frame-options-deny.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/nph-cached-csp.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/test-csp-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/check-https-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/echo-https-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/post-https-header.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-alert-fail.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-fail.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image-on-HTTP-0.9-default-port-page-allowed.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image-on-HTTP-0.9-page-blocked.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-sandbox-should-not-persist-on-navigation.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-worker-fail.pl:
* LayoutTests/http/tests/security/originHeader/resources/print-origin.cgi:
* LayoutTests/http/tests/security/resources/escape-form-data-field-names.cgi:
* LayoutTests/http/tests/ssl/resources/no-http-referer.cgi:
* LayoutTests/http/tests/webfont/slow-ahem-loading.cgi:
* LayoutTests/http/tests/websocket/tests/hybi/httponly-cookie.pl:
* LayoutTests/http/tests/websocket/tests/hybi/secure-cookie-insecure-connection.pl:
* LayoutTests/http/tests/websocket/tests/hybi/secure-cookie-secure-connection.pl:
* LayoutTests/http/tests/xmlhttprequest/hello-world.cgi:
* LayoutTests/http/tests/xmlhttprequest/interactive-state.cgi:
* LayoutTests/http/tests/xmlhttprequest/methods.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/304.pl:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-access-control-origin-header.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-no-credentials.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-star.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allowlist-request-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allowlist-response-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-denied.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-get-fail-non-simple.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-non-get-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-options-not-supported.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-post-fail-non-simple.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-allow-origin-null.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-denied-without-wildcard.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-denied.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-site-progress-events.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-set-cookie.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-utf-8.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-windows-1251.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/noContentLength.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo-as-ascii.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo-as-utf-8.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/print-cache-control-header.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/print-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/small-chunks.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/001-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/003-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/004-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/005-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/010-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/013-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/015-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/workers/resources/methods.cgi:
* LayoutTests/platform/wincairo/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/269838@main">https://commits.webkit.org/269838@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/82d58f7db4c8ff1db8f517bb070a837a14d3a48a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/23713 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/1825 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/24830 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/25866 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/21885 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/3415 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/24241 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/22429 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/23956 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/1374 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/20514 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/26462 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/1158 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/21420 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/27678 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/21630 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/21690 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/25445 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/1129 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/18826 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/1112 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5676 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/1541 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/1424 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rkirsling commented on LayoutTests/platform/wincairo/TestExpectations:
```diff
@@ -896,8 +896,6 @@ http/tests/xmlhttprequest/cross-origin-authorization.html [ Failure ]
 http/tests/xmlhttprequest/cross-origin-cookie-storage.html [ Failure ]
 http/tests/xmlhttprequest/cross-origin-no-authorization.html [ Failure ]
 http/tests/xmlhttprequest/logout.html [ Failure ]
-http/tests/xmlhttprequest/origin-header-same-origin-get-async.html [ Failure ]
-http/tests/xmlhttprequest/origin-header-same-origin-get-sync.html [ Failure ]
```
This only fixes two tests (which it looks like both depend on access-control-basic-allow-access-control-origin-header.cgi)? 🤔 Are problems observed with the rest of the test files here?

fujii replied:
I don't know how many tests pass by this change. WinCairo still skips a lot of tests under http/tests. The one of the reasons is this text mode problem. I have to check other test results after fixing Perl and Python CGI problems.

rkirsling reviewed (approved):
Seems sensible and everything's looking green, so r=me.

fujii committed:
STDIN and STDOUT of Perl CGI scripts should be binary mode for Windows Perl
https://bugs.webkit.org/show_bug.cgi?id=263701

Reviewed by Ross Kirsling.

Windows Perl converts '\n' to '\r\n' in text mode. STDIN and STDOUT of
Perl CGI scripts have to be binary mode for Windows Perl. "use CGI"
automatically does it. However, "use CGI" is not recomended nowadays,
and it's inefficient to "use CGI" just for setting binmode for them.
Added "binmode STDOUT" for Perl CGI scripts not using CGI module.
Also, added "binmode STDIN" for scripts reading STDIN.

* LayoutTests/http/tests/cache/resources/current-time.cgi:
* LayoutTests/http/tests/cache/resources/echo-cacheable.cgi:
* LayoutTests/http/tests/cache/resources/echo-no-store.cgi:
* LayoutTests/http/tests/cache/resources/random-cached-iframe.cgi:
* LayoutTests/http/tests/cache/resources/random-cached.cgi:
* LayoutTests/http/tests/cache/resources/random-max-age.cgi:
* LayoutTests/http/tests/cache/resources/random.cgi:
* LayoutTests/http/tests/cache/resources/subresource-failover-to-network.cgi:
* LayoutTests/http/tests/cookies/resources/clearCookies.cgi:
* LayoutTests/http/tests/cookies/resources/getCookies.cgi:
* LayoutTests/http/tests/cookies/resources/setCookies.cgi:
* LayoutTests/http/tests/history/redirect-200-refresh-0-seconds.pl:
* LayoutTests/http/tests/history/redirect-200-refresh-2-seconds.pl:
* LayoutTests/http/tests/history/resources/redirect-helper.pl:
* LayoutTests/http/tests/incremental/pause-in-script-element.pl:
* LayoutTests/http/tests/incremental/resources/slow-utf8-css.pl:
* LayoutTests/http/tests/incremental/slow-utf8-html.pl:
* LayoutTests/http/tests/incremental/slow-utf8-text.pl:
* LayoutTests/http/tests/incremental/split-hex-entities.pl:
* LayoutTests/http/tests/inspector/network/resources/cors-data.pl:
* LayoutTests/http/tests/links/resources/redirect-helper.pl:
* LayoutTests/http/tests/loading/onload-vs-immediate-refresh.pl:
* LayoutTests/http/tests/loading/resources/404-with-empty-body-no-content-type.cgi:
* LayoutTests/http/tests/loading/resources/404-with-empty-body.cgi:
* LayoutTests/http/tests/local/slow-css-pass.cgi:
* LayoutTests/http/tests/mime/resources/style-with-empty-content-type.cgi:
* LayoutTests/http/tests/misc/BOM-override.pl:
* LayoutTests/http/tests/misc/error404.pl:
* LayoutTests/http/tests/misc/nph-non-utf8-header-name.pl:
* LayoutTests/http/tests/misc/resources/BOM-override-2.pl:
* LayoutTests/http/tests/misc/resources/BOM-override-3.pl:
* LayoutTests/http/tests/misc/resources/image-slow.pl:
* LayoutTests/http/tests/misc/resources/intrinsic-ratio-slow.pl:
* LayoutTests/http/tests/misc/resources/percent-sign-in-form-data-field-name.cgi:
* LayoutTests/http/tests/misc/resources/post-echo-and-notify-done.cgi:
* LayoutTests/http/tests/misc/resources/post-echo.cgi:
* LayoutTests/http/tests/misc/resources/script-slow1.pl:
* LayoutTests/http/tests/misc/resources/script-slow2.pl:
* LayoutTests/http/tests/misc/resources/slow-async-script.cgi:
* LayoutTests/http/tests/misc/resources/slow-defer-script.cgi:
* LayoutTests/http/tests/misc/resources/slow-stylesheet.cgi:
* LayoutTests/http/tests/misc/resources/svg-slow.pl:
* LayoutTests/http/tests/misc/resources/text-refresh.pl:
* LayoutTests/http/tests/misc/resources/uncacheable-script.cgi:
* LayoutTests/http/tests/misc/xmltokenizer-do-not-crash.pl:
* LayoutTests/http/tests/navigation/resources/error404.pl:
* LayoutTests/http/tests/navigation/resources/redirect-cycle-1.pl:
* LayoutTests/http/tests/navigation/resources/redirect-cycle-2.pl:
* LayoutTests/http/tests/navigation/resources/redirect-to-referrer-check.pl:
* LayoutTests/http/tests/navigation/resources/redirect302-metaredirect.pl:
* LayoutTests/http/tests/navigation/resources/redirect302.pl:
* LayoutTests/http/tests/navigation/resources/reload-subframe-content.pl:
* LayoutTests/http/tests/navigation/resources/reload-subframe.pl:
* LayoutTests/http/tests/navigation/resources/response204.pl:
* LayoutTests/http/tests/navigation/resources/slow-resource-1-sec.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin-cacheable.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin-unknown-type.pl:
* LayoutTests/http/tests/plugins/resources/mock-plugin.pl:
* LayoutTests/http/tests/plugins/resources/plugin-document-has-focus.pl:
* LayoutTests/http/tests/plugins/resources/slow-resource.pl:
* LayoutTests/http/tests/preload/resources/nph-invalid_resources_from_header.pl:
* LayoutTests/http/tests/resources/post-and-verify-hybrid.cgi:
* LayoutTests/http/tests/resources/post-and-verify.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/nph-cached-xfo.pl:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-allowall.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-deny.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-frame-ancestors-same-origin-allow.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-frame-ancestors-same-origin-deny.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-invalid.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-multiple-headers-conflict.cgi:
* LayoutTests/http/tests/security/XFrameOptions/resources/x-frame-options-multiple-headers-sameorigin.cgi:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/frame-ancestors-self-x-frame-options-deny.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/nph-cached-csp.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/resources/test-csp-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/check-https-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/echo-https-header.pl:
* LayoutTests/http/tests/security/contentSecurityPolicy/upgrade-insecure-requests/resources/post-https-header.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-alert-fail.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-fail.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image-on-HTTP-0.9-default-port-page-allowed.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image-on-HTTP-0.9-page-blocked.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-image.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-sandbox-should-not-persist-on-navigation.pl:
* LayoutTests/http/tests/security/http-0.9/resources/nph-worker-fail.pl:
* LayoutTests/http/tests/security/originHeader/resources/print-origin.cgi:
* LayoutTests/http/tests/security/resources/escape-form-data-field-names.cgi:
* LayoutTests/http/tests/ssl/resources/no-http-referer.cgi:
* LayoutTests/http/tests/webfont/slow-ahem-loading.cgi:
* LayoutTests/http/tests/websocket/tests/hybi/httponly-cookie.pl:
* LayoutTests/http/tests/websocket/tests/hybi/secure-cookie-insecure-connection.pl:
* LayoutTests/http/tests/websocket/tests/hybi/secure-cookie-secure-connection.pl:
* LayoutTests/http/tests/xmlhttprequest/hello-world.cgi:
* LayoutTests/http/tests/xmlhttprequest/interactive-state.cgi:
* LayoutTests/http/tests/xmlhttprequest/methods.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/304.pl:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-access-control-origin-header.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-no-credentials.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow-star.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allowlist-request-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-allowlist-response-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-denied.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-get-fail-non-simple.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-non-get-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-options-not-supported.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-basic-post-fail-non-simple.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-allow-origin-null.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-allow.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-denied-without-wildcard.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/access-control-sandboxed-iframe-denied.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/cross-site-progress-events.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-set-cookie.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-utf-8.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/get-windows-1251.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/noContentLength.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo-as-ascii.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo-as-utf-8.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/post-echo.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/print-cache-control-header.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/print-headers.cgi:
* LayoutTests/http/tests/xmlhttprequest/resources/small-chunks.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/001-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/003-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/004-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/005-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/010-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/013-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/web-apps/015-test.cgi:
* LayoutTests/http/tests/xmlhttprequest/workers/resources/methods.cgi:
* LayoutTests/platform/wincairo/TestExpectations:

Canonical link: https://commits.webkit.org/269838@main
